### PR TITLE
Vet Clinic Database: Add "Join Table" for Visits PR

### PR DESCRIPTION
**Pull request by WalterOkumu**
## Requirements
- Write queries to answer the following:
   - Who was the last animal seen by William Tatcher?
      - **Result:** 
         - Query_26

   - How many different animals did Stephanie Mendez see?
      - **Result:** 
         - Query_27

   - List all vets and their specialties, including vets with no specialties.
      - **Result:** 
         - Query_28

   - List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
      - **Result:** 
         - Query_29

   - What animal has the most visits to vets?
      - **Result:** 
         - Query_30

   - Who was Maisy Smith's first visit?
      - **Result:** 
         - Query_31

   - Details for most recent visit: animal information, vet information, and date of visit.
      - **Result:** 
         - Query_32

   - How many visits were with a vet that did not specialize in that animal's species?
      - **Result:**
         - Query_33

   - What specialty should Maisy Smith consider getting? Look for the species she gets the most.
      - **Result:** 
         - Query_34

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -43,4 +43,32 @@ ADD COLUMN owner_id int;
...
 ALTER TABLE animals
 ADD FOREIGN KEY (owner_id) REFERENCES owners(id)
-ON DELETE CASCADE;
\ No newline at end of file
+ON DELETE CASCADE;
+
+-- Create a table named vets
+CREATE TABLE vets (
+	id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	name TEXT,
```
Comment (sja-thedude) [inline schema.sql:51]: - [ ] Kindly make sure that name has a varchar data type.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @WalterOkumu, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (WalterOkumu) [reply schema.sql:51]: Hi @sja-thedude

Please advise why.

From my research, using VARCHAR & TEXT without the max length is exactly the same, therefore there is no performance advantage. [Read More](https://www.geeksforgeeks.org/postgresql-difference-between-char-varchar-and-text/)

**Review by uwadonat — APPROVED** [review]
Hi @WalterOkumu,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Highlights
- Well organized project ✔️
image2


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**